### PR TITLE
WebTracer now supports http/protobuf exporter

### PR DESCRIPTION
**Pull request by scheler**
**Preview**: https://deploy-preview-3133--opentelemetry.netlify.app/docs/instrumentation/js/exporters/

### CODE REVIEW

**Review by cartermp — APPROVED** [review]
Interesting, wasn't aware it was supported now. Text looks good modulo the spelling error.

**Comment by scheler** [comment]
> Interesting, wasn't aware it was supported now. Text looks good modulo the spelling error.

yes, @pkanal added it few months ago.

File: content/en/docs/instrumentation/js/exporters.md [hunk 1]
```diff
@@ -140,15 +140,16 @@ Below you will find instructions to use the right exporter, to configure your
...
 
-#### Use OTLP exporter with HTTP/JSON
+#### Use OTLP exporter with HTTP/JSON or HTTP/protobuf
 
-[OpenTelemetry Collector Exporter with gRPC][] and [OpenTelemetry Collector
-Exporter with protobuf][] do only work with Node.js, therefore you are limited
-to use the [OpenTelemetry Collector Exporter with HTTP][].
+[OpenTelemetry Collector Exporter with gRPC][] works only with Node.js,
+therefore you are limited to use the [OpenTelemetry Collector Exporter with
+HTTP/JSON][] or [OpenTelemetry Collector Exporter with HTTP/protobuf][]
```
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:147]:
```suggestion
HTTP/JSON][] or [OpenTelemetry Collector Exporter with HTTP/protobuf][].
```

File: content/en/docs/instrumentation/js/exporters.md [hunk 2]
```diff
@@ -140,15 +140,16 @@ Below you will find instructions to use the right exporter, to configure your
...
 
 Make sure that the receiving end of your exporter (collector or observability
-backend) does support `http/json`, and that you are exporting your data to the
-right endpoint, i.e., make sure that your port is set to `4318`.
+backend) does accept `http/json` if you are using [OpenTelemetry Collector
```
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:150]:
```suggestion
backend) accepts `http/json` if you are using [OpenTelemetry Collector
```

File: content/en/docs/instrumentation/js/exporters.md [hunk 3]
```diff
@@ -140,15 +140,16 @@ Below you will find instructions to use the right exporter, to configure your
...
+Exporter with HTTP/JSON][], and that you are exporting your data to the right
+endpoint, i.e., make sure that your port is set to `4318`.
```
Comment (chalin) [inline content/en/docs/instrumentation/js/exporters.md:152]:
```suggestion
endpoint with your port set to 4318.
```

**Review by chalin — APPROVED** [review]
See inline comments for minor copyedits.

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
That's awesome!

**Pull request merged**